infra/app-flask/service/outputs.tf

```diff
@@ -30,3 +30,36 @@ output "service_endpoint" {
 output "service_name" {
   value = local.service_name
 }
+
```

Comment:
I don't think any of these need to be output from the service? We don't need to look them up outside of the service module for anything (at least at the moment) right?